examples/audio_rttl/Makefile

```diff
@@ -33,10 +33,8 @@
 #
 ############################################################################
 
--include $(TOPDIR)/.config
--include $(TOPDIR)/Make.defs
+include $(TOPDIR)/Make.defs
 -include $(SDKDIR)/Make.defs
-include $(APPDIR)/Make.defs
```

Comment:
This makefile needs DELIM, it gets it from apps/Make.defs.  I'm not sure if there is any indirect definition somehow.

Reply:
$(TOPDIR)/Make.defs->$(TOPDIR)/tools/Config.mk->DELIM define here.

Reply:
BTW, there is only one DELIM definition now.